test/bookmark-components/bookmarksToolbarTest.js

```diff
@@ -55,7 +55,7 @@ describe('bookmarksToolbar', function () {
         .waitForVisible('[data-test-id="bookmarkToolbarButton"][title=demo1]')
         .click(bookmarksToolbar)
         .click('[data-test-id="bookmarkToolbarButton"][title=demo1]')
-        .waitForVisible('.contextMenuItemText[data-l10n-id=emptyFolderItem]')
+        .waitForVisible('[data-test-id="contextMenuItemText"][data-l10n-id=emptyFolderItem]')
```

Comment:
Can we maybe use variables here as well?

Like: `.waitForVisible(``${contextMenuItemText}[data-l10n-id=emptyFolderItem]``)`

Reply:
Locally I've replaced al CSS and l10n selectors (#8735) and that will be addressed with / after it

Reply:
not sure that I follow. What I was suggesting is that we use variable, so that if data-test-id is changed we can only change variable and everything will work

Reply:
what I'm saying is that we will replace the variables all together.

Reply:
I opened an issue for that task: https://github.com/brave/browser-laptop/issues/11574

Reply:
ok sounds good